tests/network-setups/fabric/dev/scripts/setupAppCCLocal.sh

```diff
@@ -12,30 +12,8 @@ if [ -d "${CHAINCODE_PATH}/${CC_NAME}" ]; then
     echo "Deleting existing ${CC_NAME} chaincode folder to copy the latest..."
     rm -rf ${CHAINCODE_PATH}/${CC_NAME}
 fi
+
+(cd ${APP_CC_PATH} && make run-vendor)
```

Comment:
Why do you need the parantheses?

Reply:
parantheses makes sure when the command ends, we are back to original directory, and not to the one where did `cd` to.